nixos/modules/services/networking/ntpd.nix

```diff
@@ -36,9 +40,34 @@ in
       enable = mkOption {
         default = false;
         description = ''
-          Whether to synchronise your machine's time using the NTP
-          protocol.
+          Whether to synchronise your machine's time using ntpd.
+
+          Disables systemd.timesyncd if enabled.
+        '';
+      };
+
+      restrictDefault = mkOption {
+        type = types.listOf types.str;
+        description = ''
+          The restriction flags to be set by default.
+
+          The default flags prevent external hosts from using ntpd as a DDoS
+          reflector, seting system time, and querying OS/ntpd version.
```

Comment:
```suggestion
          reflector, setting system time, and exposing OS/NTPD version.
```

Reply:
I fixed the typo on `setting`, but I believe the other two words should stay as they were.